.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+sudo: false
+node_js:
+  - 6
+os:
+  - linux
+  - osx
+before_install:
+  - pip install --user cpplint
```

Comment:
No worries, just wanted to make sure we don't forget about this.
Oh, right – I forgot we don't have `sudo` in the CI containers